CRITICAL - 2023-09-18 12:59:29 --> Call to undefined function localhost/index8()
in APPPATH\Views\index.php on line 62.
 1 SYSTEMPATH\View\View.php(224): include()
 2 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(9): view('index')
 5 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
ERROR - 2023-09-18 13:14:10 --> mysqli_sql_exception: Unknown column 'password' in 'field list' in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#7 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-09-18 13:14:10 --> Unknown column 'password' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (NULL, \'123\', \'123\', \'\', \'user\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (NULL, \'123\', \'123\', \'\', \'user\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (NULL, \'123\', \'123\', \'\', \'user\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (:name:, :password:, :email:, :id:, :role:, :is_active:)', [...], false)
 5 APPPATH\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\reg->signup()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
ERROR - 2023-09-18 13:14:19 --> mysqli_sql_exception: Unknown column 'password' in 'field list' in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#7 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-09-18 13:14:19 --> Unknown column 'password' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (NULL, \'123\', \'123\', \'\', \'user\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (NULL, \'123\', \'123\', \'\', \'user\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (NULL, \'123\', \'123\', \'\', \'user\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`name`, `password`, `email`, `id`, `role`, `is_active`) VALUES (:name:, :password:, :email:, :id:, :role:, :is_active:)', [...], false)
 5 APPPATH\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\reg->signup()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
ERROR - 2023-09-18 13:19:55 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', '7a_database', 3306, '', 0)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#7 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-09-18 13:19:55 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (:id:, :name:, :pass:, :email:, :role:, :is_active:)', [...], false)
 3 APPPATH\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\reg->signup()
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
ERROR - 2023-09-18 13:20:17 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', '7a_database', 3306, '', 0)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#7 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2023-09-18 13:20:17 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (:id:, :name:, :pass:, :email:, :role:, :is_active:)', [...], false)
 3 APPPATH\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\reg->signup()
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
ERROR - 2023-09-18 13:20:44 --> mysqli_sql_exception: Column 'name' cannot be null in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#7 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-09-18 13:20:44 --> Column 'name' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (\'\', NULL, \'123\', \'123@123.123\', \'user\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (\'\', NULL, \'123\', \'123@123.123\', \'user\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (\'\', NULL, \'123\', \'123@123.123\', \'user\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (:id:, :name:, :pass:, :email:, :role:, :is_active:)', [...], false)
 5 APPPATH\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\reg->signup()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
ERROR - 2023-09-18 13:21:04 --> mysqli_sql_exception: Column 'name' cannot be null in C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\index8\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\index8\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\index8\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\index8\app\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\index8\system\CodeIgniter.php(932): App\Controllers\reg->signup()
#6 C:\xampp\htdocs\index8\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
#7 C:\xampp\htdocs\index8\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\index8\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\index8\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-09-18 13:21:04 --> Column 'name' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (\'\', NULL, \'123\', \'123@123.123\', \'user\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (\'\', NULL, \'123\', \'123@123.123\', \'user\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (\'\', NULL, \'123\', \'123@123.123\', \'user\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`id`, `name`, `pass`, `email`, `role`, `is_active`) VALUES (:id:, :name:, :pass:, :email:, :role:, :is_active:)', [...], false)
 5 APPPATH\Controllers\reg.php(22): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\reg->signup()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\reg))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\index8\\public\\index.php')
